server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -449,6 +449,26 @@ public void consumerExists(
         }
     }
 
+    @ApiOperation(
+        notes = "Checks for the existence of a Consumer in bulk. This API returns uuids of non-existing" +
+        "consumer",
+        value = "")
+    @ApiResponses({ @ApiResponse(code = 400, message = "Return 404 when empty or null array is passed"),
```

Comment:
These response codes don't line up with the actual output in the code below, nor the outputs defined in the task.

According to the Jira task, we are looking for two things:
- If the list contains any consumer UUIDs which do not exist, this endpoint should return a 404 and a list containing those consumer UUIDs which did not exist.
- If the list contains only existing consumer UUIDs, this endpoint should return a 204 and an empty list. Note that an empty input list matches this case.

Reply:
The Response to API does not add up to Jira, as I believe it was written according to HEAD API to be as a reference point.

Reply:
At this point I am unsure if `BAD_REQUEST` is needed or not.